### PR TITLE
Add http cache headers to headless website controller

### PR DESCRIPTION
**Pull request by drakulis**
closes https://github.com/sulu/SuluHeadlessBundle/issues/50

### CODE REVIEW

File: Controller/HeadlessWebsiteController.php [hunk 1]
```diff
@@ -49,13 +50,14 @@ public function indexAction(
             return $this->renderTemplateResponse($structure, $requestFormat, $preview, $data);
```
[the remark this answers is not in the document]
Comment (drakulis) [reply Controller/HeadlessWebsiteController.php]: done

File: Controller/HeadlessWebsiteController.php [hunk 2]
```diff
@@ -49,13 +50,14 @@ public function indexAction(
...
 
-        return new Response(
-            $json,
-            200,
-            [
-                'Content-Type' => 'application/json',
-            ]
-        );
+        $response = new Response($json);
+        $response->headers->set('Content-Type', 'application/json');
```
[the remark this answers is not in the document]
Comment (drakulis) [reply Controller/HeadlessWebsiteController.php]: done

**Comment by alexander-schranz** [comment]
@drakulis Thank you!

**Comment by drakulis** [comment]
@alexander-schranz there were some unit tests which were failing 😨

**Comment by alexander-schranz** [comment]
@drakulis they were not your faults, we testing against sulu master branch here and there where some changes in the new segments feature. Don't wanted that you have to fight with that and created a own PR to fix them: #52

**Comment by drakulis** [comment]
ahhh.. ok. great.

**Comment by nnatter** [comment]
Thanks a lot for fixing this!

**Pull request merged**